### PR TITLE
Dev adelacecilia lets connect ce a mai ramas plus skills

### PR DESCRIPTION
**Pull request by adelacecilia**

### CODE REVIEW

**Review by QuintyHH — APPROVED** [review]
Some empty lines, but otherwise ok :)
Approved

File: src/_component/about-me/about-me.scss [hunk 1]
```diff
@@ -9,69 +9,49 @@
   display: flex;
   justify-content: center;
   align-items: center;
-  font-size: 13px;
+  font-size: 10px;
   font-weight: bold;
+
+  i {
```
Comment (marianzburlea) [inline src/_component/about-me/about-me.scss]: bad selector :) let's chat

File: src/_component/skills/skills.scss [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+.skills__label {
+  line-height: 48px;
+  color: #666666;
+}
+
+.skills__label {
+  flex-grow: 1;
+}
+
```
Comment (marianzburlea) [inline src/_component/skills/skills.scss]: Why two times? Refactor to only one selector declaration.

**Review by marianzburlea — CHANGES_REQUESTED** [review]
💯

File: src/_component/skills/skills.scss (hunk 2 above)
Comment (adelacecilia) [reply src/_component/skills/skills.scss]: Done! :)

File: src/_component/about-me/about-me.scss (hunk 1 above)
Comment (adelacecilia) [reply src/_component/about-me/about-me.scss]: Sorted! :D

**Review by QuintyHH — APPROVED** [review]
:100:

**Review by marianzburlea — APPROVED** [review]
💯

**Pull request merged**